### ISSUE TITLE
[Victory] Apply match button without choosing a match throw an error

### ISSUE THREAD

**Issue by saman** [eMoflon/emoflon-ibex-ui#152]
If I click on `Apply match` button without choosing a match, it crashes the UI and throws this error:

```
Exception in thread "IbexDebugUI - SWT UI thread" java.lang.ArrayIndexOutOfBoundsException: 0
	at org.emoflon.ibex.tgg.ui.debug.views.MatchListView$2.widgetSelected(MatchListView.java:69)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:249)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:86)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4247)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1508)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1531)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1516)
	at org.eclipse.swt.widgets.Widget.notifyListeners(Widget.java:1320)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4091)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3712)
	at org.emoflon.ibex.tgg.ui.debug.core.IbexDebugUI.run(IbexDebugUI.java:69)
	at org.emoflon.ibex.tgg.run.debug.MODELGEN_App.main(MODELGEN_App.java:59)
```

**Comment by Ayurshi** [comment]
Also happens when Apply Match is double clicked(by mistake) same exception is thrown

**Comment by saman** [comment]
> Also happens when Apply Match is double clicked(by mistake) same exception is thrown

I think is the same behavior, because when you click the list will be refreshed and also again there is no match selected, and cause the same error again.

**Comment by LegionaryCohort** [comment]
Fixed in commit bc8df85
